tests/systemtests/bank_test.go

```diff
@@ -0,0 +1,252 @@
+				assert.True(t, found)
+				RequireTxSuccess(t, txResult)
+			}
+		})
+	}
+
+	// test tx bank send with insufficient funds
+	insufficientCmdArgs := bankSendCmdArgs[0 : len(bankSendCmdArgs)-1]
+	insufficientCmdArgs = append(insufficientCmdArgs, initialBalance, "--fees=10stake")
+	rsp := cli.Run(insufficientCmdArgs...)
+	RequireTxFailure(t, rsp)
+	assert.Contains(t, rsp, sdkerrors.ErrInsufficientFunds.Error())
+
+	// test tx bank send with unauthorized signature
+	assertUnauthorizedErr := func(t assert.TestingT, gotErr error, gotOutputs ...interface{}) bool {
+		assert.Len(t, gotOutputs, 1)
+		code := gjson.Get(gotOutputs[0].(string), "code")
+		assert.True(t, code.Exists())
+		assert.Equal(t, int64(sdkerrors.ErrUnauthorized.ABCICode()), code.Int())
+		return false
+	}
+	invalidCli := cli
+	invalidCli.chainID = cli.chainID + "a" // set invalid chain-id
+	rsp = invalidCli.WithRunErrorMatcher(assertUnauthorizedErr).Run(bankSendCmdArgs...)
+	RequireTxFailure(t, rsp)
+
+	// test tx bank send generate only
+	assertGenOnlyOutput := func(t assert.TestingT, gotErr error, gotOutputs ...interface{}) bool {
+		assert.Len(t, gotOutputs, 1)
+		rsp := gotOutputs[0].(string)
+		// get msg from output
+		msgs := gjson.Get(rsp, "body.messages").Array()
+		assert.Len(t, msgs, 1)
+		// check from address is equal to account1 address
+		fromAddr := gjson.Get(msgs[0].String(), "from_address").String()
+		assert.Equal(t, account1Addr, fromAddr)
+		// check to address is equal to account2 address
+		toAddr := gjson.Get(msgs[0].String(), "to_address").String()
+		assert.Equal(t, account2Addr, toAddr)
+		return false
+	}
+	genCmdArgs := append(bankSendCmdArgs, "--generate-only")
+	_ = cli.WithRunErrorMatcher(assertGenOnlyOutput).Run(genCmdArgs...)
+
+	// test tx bank send with dry-run flag
+	assertDryRunOutput := func(t assert.TestingT, gotErr error, gotOutputs ...interface{}) bool {
+		assert.Len(t, gotOutputs, 1)
+		rsp := gotOutputs[0].(string)
+		// check gas estimate value found in output
+		assert.Contains(t, rsp, "gas estimate")
+		return false
+	}
+	dryRunCmdArgs := append(bankSendCmdArgs, "--dry-run")
+	_ = cli.WithRunErrorMatcher(assertDryRunOutput).Run(dryRunCmdArgs...)
+}
+
+func TestBankMultiSendTxCmd(t *testing.T) {
+	// scenario: test bank multi-send command
+	// given a running chain
+
+	sut.ResetChain(t)
+	cli := NewCLIWrapper(t, sut, verbose)
+	// add genesis account with some tokens
+	account1Addr := cli.AddKey("account1")
+	account2Addr := cli.AddKey("account2")
+	account3Addr := cli.AddKey("account3")
+	require.NotEqual(t, account1Addr, account2Addr, account3Addr)
+	denom := "stake"
+	initialAmount := 10000000
+	initialBalance := fmt.Sprintf("%d%s", initialAmount, denom)
+	sut.ModifyGenesisCLI(t,
+		[]string{"genesis", "add-genesis-account", account1Addr, initialBalance},
+		[]string{"genesis", "add-genesis-account", account2Addr, initialBalance},
+		[]string{"genesis", "add-genesis-account", account3Addr, initialBalance},
```

Comment:
would it make sense to not add account3 to genesis so that it would be created by the submission? Then this scenario is handled as well